src/msgwindow.c

```diff
@@ -315,11 +317,13 @@ static const GdkColor *get_color(gint msg_color)
 
 
 /**
- *  Adds a new message in the compiler tab treeview in the messages window.
+ * Adds a new message in the compiler tab treeview in the messages window.
  *
- *  @param msg_color A color to be used for the text. It must be an element of #MsgColors.
- *  @param format @c printf()-style format string.
- *  @param ... Arguments for the @c format string.
+ * @param msg_color A color to be used for the text. It must be an element of #MsgColors.
+ * @param format    @c printf()-style format string.
+ * @param ...       Arguments for the @c format string.
+ *
```

Comment:
Could be interesting to add a `@see msgwin_compiler_add_string()`, wouldn't it?  And similar for the other functions.  Just a suggestion though, but could avoid a little confusion for the C API users.